LocationSync/LocationSync.h

```diff
@@ -151,10 +151,12 @@ namespace Plugin {
                 : Interval(30)
                 , Retries(8)
                 , Source()
+                , TimeZoneOverrideFile()
             {
                 Add(_T("interval"), &Interval);
                 Add(_T("retries"), &Retries);
                 Add(_T("source"), &Source);
+                Add(_T("timeZoneOverrideFile"), &TimeZoneOverrideFile);
```

Comment:
field names in JSON for the config are typically all lowercase. suggest calling this timezonefile ?